pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+_Please tick all applicable boxes, fill in the **bold** fields, and remove these instructions from your PR._
```

Comment:
Great thinking on the `_` vs HTML comments 👍 I will carry that out.

I don't know about your other suggestion. Once they have submitted the PR, mails go out to all watchers with incomplete information, which is undesirable IMHO. So that why I dislike it and will not change it unless I'm a minority that has that opinion.

I fully agree that ticking the boxes is easier from the interface than from GFM, but I think the time gain is negligible since:
 - Most people only submit 1 PR ever (for their own list), and even if you would look at the [top committers](https://github.com/sindresorhus/awesome/graphs/contributors), the distribution of number of commits drops off very steeply: 70, 18, 11, 6, 4, 3 (x5), 2 (x20), 1 (all the rest) and is evidence for the same phenomenon.
 - The difference in time (between clicking and typing) is going to be 5 seconds tops (probably just 2 seconds). If that is too much, I honestly doubt that the person involved is able to maintain a high-quality awesome list in his very limited time.